packages/csv-parser-order/test/add-return-info.spec.js

```diff
@@ -0,0 +1,187 @@
+import fs from 'fs'
+import StreamTest from 'streamtest'
+import path from 'path'
+
+import AddReturnInfoParser from '../src/parsers/add-return-info'
+
+describe('AddReturnInfoParser', () => {
+  it('should be a class', () => {
+    expect(typeof AddReturnInfoParser).toBe('function')
```

Comment:
I removed it because this variable is used in other tests so if there is anything wrong with it, it will be found in other tests.